dluhc-component-library/src/stories/Map.stories.tsx

```diff
@@ -16,6 +17,7 @@ const MapComponent = ({ lat, lng, zoom }: MapComponentProps) => {
         zoom={zoom}
         style={{ height: "500px", width: "500px" }}
       />
+      <DrawingLayer />
```

Comment:
It might be worth just having a new story for drawing rather than putting it on each one?